src/main/java/com/spotinst/sdkjava/model/bl/ocean/kubernetes/ClusterDynamicVolumeSize.java

```diff
@@ -0,0 +1,101 @@
+package com.spotinst.sdkjava.model.bl.ocean.kubernetes;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+
+import java.util.HashSet;
+import java.util.Set;
+
+public class ClusterDynamicVolumeSize {
```

Comment:
Should follow consistent classname convention.

Reply:
I reuse the class from the cluster creation, will create another one for VNG